app/containers/ViewQuestion/Question.jsx

```diff
@@ -1,11 +1,22 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
 import * as routes from 'routes-config';
 
 import Content from './Content';
-import { QUESTION_TYPE } from './constants';
+import { QUESTION_TYPE, EXPERTS, QUESTIONS, TUTORIALS } from './constants';
 
 export const Question = props => {
+  const { postType, id } = props.questionData;
+
+  const type = useMemo(
+    () => {
+      if (postType === 0) return EXPERTS;
```

Comment:
This `type` value is needed for routing only. Move this mapping to `routes-config.js`. Pass`postType` as number to  `questionEdit(...)`.  This way we will keep routing related code all together in a single place.

Reply:
custom hook or reusable function outside of this component would be fine